integration/tests/cook/test_basic.py

```diff
@@ -110,12 +112,20 @@ def test_disable_mea_culpa(self):
             return
         uuid, resp = util.submit_job(self.cook_url, command='sleep 30', env={'EXECUTOR_TEST_EXIT': '1'},
                                      disable_mea_culpa_retries=True)
-        job = util.wait_for_job(self.cook_url, uuid, 'completed')
-        self.assertEqual(job['state'], 'failed', json.dumps(job, indent=2))
-        self.assertEqual(job['retries_remaining'], 0, json.dumps(job, indent=2))
-        instances = job['instances']
-        self.assertEqual(1, len(instances), json.dumps(job, indent=2))
-        self.assertEqual('Mesos executor terminated', instances[0]['reason_string'], json.dumps(job, indent=2))
+        try:
+            instance = util.wait_for_instance(self.cook_url, uuid)
+            if instance['executor'] != 'cook':
```

Comment:
I think this test should have a `skipIf` annotation as there is no point in running the test if Cook executor is not in use.

Reply:
I think the issue here is that it might be configured but not used for a particular instance (like in pub cloud) which you wouldn't know until running it. Maybe we can expose this via settings and then make the change?

Reply:
@pschorf What do you mean by

> we can expose this via settings

Reply:
@shamsimam and I discussed and landed on adding an environment variable, e.g. `COOK_TEST_COOK_EXECUTOR`, which can be used to skip tests like this that assume the cook executor is in use

Reply:
What about using a combo of checking if it's configured and you're not submitting via docker? Would that cover it? We already have so many environment variables.

Reply:
Good call, done (see `util.is_cook_executor_in_use`)